server/libmain.go

```diff
@@ -94,11 +77,54 @@ func StartAndRunServices(env environment.Env) {
 		capabilitiesServer := capabilities_server.NewCapabilitiesServer( /*supportCAS=*/ true /*supportRemoteExec=*/, false)
 		repb.RegisterCapabilitiesServer(grpcServer, capabilitiesServer)
 	}
+}
+
+func StartGRPCService(env environment.Env, buildBuddyServer *buildbuddy_server.BuildBuddyServer, port *int, credentialOption grpc.ServerOption) {
```

Comment:
nit: It's rare to log.Fatalf in a method so it'd be nice to call this FnOrDie so callers know it's all or nothing. 

rename to StartGRPCServiceOrDie?